Cargo.toml

```diff
@@ -50,8 +50,21 @@ cc = { git = "https://github.com/mqudsi/cc-rs", branch = "fish" }
 rsconf = { git = "https://github.com/mqudsi/rsconf", branch = "master" }
 
 [lib]
-crate-type = ["staticlib"]
+crate-type = ["rlib"]
 path = "fish-rust/src/lib.rs"
+doctest = false
+
+[[bin]]
+name = "fish"
```

Comment:
I have a slight preference for`src/fish.rs` over `src/bin/fish.rs` because the `fish` prefix already implies it's an entrypoint. Not a strong opinion though.